src/pylhe/__init__.py

```diff
@@ -238,13 +234,12 @@ def _indent(elem, level=0):
             elem.text = i + "  "
         if not elem.tail or not elem.tail.strip():
             elem.tail = i
-        for elem in elem:
-            _indent(elem, level + 1)
+        for inner_elem in elem:
+            _indent(inner_elem, level + 1)
         if not elem.tail or not elem.tail.strip():
```

Comment:
Wait, having the later elem reference the inner elem is intentional? It's part of the tests, anyway.

Reply:
I should check that carefully. it could be that the test is copied from pylhe's output to avoid breaking it.

Reply:
But since I stole it from https://stackoverflow.com/a/33956544 with support from old pythons, I think it works as desired (given the 95 up votes...).

Reply:
IMO one can just rewrite the later uses of `elem` to `inner_elem` too. Then it is at least clear that it is overridden.

Reply:
I should be able to make this explicit, let's see. It's hard to tell from the diff which is better.

Reply:
3.8 was locked and frozen as EoL today, by the way.

Reply:
```python
for inner_elem in elem:
    _indent(inner_elem, level + 1)
if not elem[-1].tail or not elem[-1].tail.strip():
    elem[-1].tail = i
```
is better, IMO.

Reply:
FYI, the logic itself is correct, though, you do want to handle the last one like this.